app/femr/ui/views/medical/edit.scala.html

```diff
@@ -30,16 +30,25 @@
     <div id="medicalContentWrap">
 
 
-        @patientOverview(viewModel.getPatientItem, viewModel.getPatientEncounterItem, "Medical")
-
-
+        @patientOverview(viewModel.getPatientItem, viewModel.getPatientEncounterItem, viewModel.getSettings, "Medical")
 
 
         <div id="mainWrap" class="backgroundForWrap">
+            @if(viewModel.getSettings.isMetric) { <!-- Alaa Serhan -->
+                <input type="hidden" id="vitalsUnits" value="metric" />
+            } else {
+                <input type="hidden" id="vitalUnits" value="imp" />
+            }
+            
             <div id="vitalsWrap">
+                @if(viewModel.getSettings.isMetric) { <!-- Alaa Serhan -->
+                    <input type="hidden" id="vitalsUnits" value="metric" />
+                } else {
+                    <input type="hidden" id="vitalUnits" value="imp" />
+                }
```

Comment:
I have to test this again and fix it. This may no longer be needed. They are referenced in js.